### PR TITLE
11846 keyboard short cuts

### PR DESCRIPTION
**Pull request by rianfowler**
Won't close the ticket but would like to get this merged into staging so Trevor can test it with a screen reader to help identify additional accessibility issues.

### CODE REVIEW

File: src/applications/edu-benefits/components/SchoolSelectField.jsx [hunk 1]
```diff
@@ -133,34 +147,50 @@ export class SchoolSelectField extends React.Component {
...
             {institutions.map(({ city, facilityCode, name, state }, index) => (
               <div key={index}>
                 <div className="radio-button">
                   <input
                     autoComplete="false"
                     checked={facilityCodeSelected === facilityCode}
-                    id={`radio-buttons-${index}`}
-                    name={name}
+                    id={`page-${currentPageNumber}-${index}`}
+                    name={`page-${currentPageNumber}`}
```
Comment (rianfowler) [inline src/applications/edu-benefits/components/SchoolSelectField.jsx]: added name and id fields that allow tab and arrowing through radio buttons

File: package.json [hunk 2]
```diff
@@ -211,7 +211,7 @@
   },
   "private": true,
   "dependencies": {
-    "@department-of-veterans-affairs/formation": "^1.3.0",
+    "@department-of-veterans-affairs/formation": "^1.3.1",
```
Comment (rianfowler) [inline package.json]: Pagination uses anchor tags without hrefs which are not added to the tab order by default. Opted for a short term fix in the design system to add this functionality and made a ticket to switch the anchor tags to buttons.

File: src/applications/edu-benefits/components/SchoolSelectField.jsx [hunk 3]
```diff
@@ -100,6 +100,7 @@ export class SchoolSelectField extends React.Component {
 
     this.props.onChange('');
     this.props.clearSearch();
+    this.searchInput.focus();
```
Comment (rianfowler) [inline src/applications/edu-benefits/components/SchoolSelectField.jsx]: Set focus to the input when clear search is activated.

File: src/applications/edu-benefits/components/SchoolSelectField.jsx [hunk 4]
```diff
@@ -173,55 +175,59 @@ export class SchoolSelectField extends React.Component {
...
-                    id={`institution-${index}-label`}
-                    htmlFor={`radio-buttons-${index}`}>
-                    <span className="institution-name">{name}</span>
-                    <span className="institution-city-state">{`${city}, ${state}`}</span>
-                  </label>
-                </div>
-              </div>))
+          <div
+            aria-live="polite"
+            aria-relevant="additions text">
```
Comment (rianfowler) [inline src/applications/edu-benefits/components/SchoolSelectField.jsx]: aria tags for a live updating section. This has to be rendered when the component is mounted

**Review by cehsu — APPROVED** [review]

**Review by erikphansen — APPROVED** [review]

**Pull request merged**